react/features/polls/components/AbstractPollAnswer.tsx

```diff
@@ -40,23 +36,25 @@ export type AbstractProps = {
  * @param {React.AbstractComponent} Component - The concrete component.
  * @returns {React.AbstractComponent}
  */
-const AbstractPollAnswer = (Component: ComponentType<AbstractProps>) => (props: InputProps) => {
+const AbstractPollAnswer = (Component: ComponentType<AbstractProps>) => (props: AbstractProps) => {
```

Comment:
please revert this - I think the new props (setCreateMode, ...) should go on InputProps type instead  - AbastractProps seem to be defined for the <Component component.